Library/Formula/sassc.rb

```diff
@@ -20,7 +20,7 @@ class Sassc < Formula
   depends_on "libsass"
 
   def install
-    ENV["SASSC_VERSION"] = "3.1.0"
+    ENV["SASSC_VERSION"] = "3.2.0"
```

Comment:
This get eval'd during the compilation and becomes the version string returned by `sassc --version`. It's how we get around hard coding versions in the sassc/libsass source. If this isn't set we infer a version string from git.